src/components/CohortBuilder/Categories/ActionCategory.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
+
+import Column from 'uikit/Column';
+
+export default class ActionCategory extends React.PureComponent {
```

Comment:
Should be a stateless component ?

Reply:
It is. Inheriting `React.PureComponent` makes it a stateless component.